app/konnect/dev-portal/audit-logging/webhook.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Set up an audit log webhook for Dev Portal
+content_type: how-to
+badge: enterprise
+---
+
+You can use the {{site.konnect_short_name}} UI or the Audit Logs API to configure webhooks for [audit logging](/konnect/dev-portal/audit-logging/). 
+
+{:.note}
+> **Note:** Currently, Dev Portal audit logs only support authorization logs, which are triggered when a user logs in to Dev Portal.
+
+{% include_cached /md/konnect/audit-logging/webhook-overview-prereq-siem-config.md %}
+
+
+## Create a webhook
+
+{% navtabs %}
+{% navtab Konnect UI %}
+1. From the navigation menu, open {% konnect_icon organizations %} **Organization**, then **Audit Logs Setup** and **Destinations**
+1. Fill in the fields in the **Destinations** tab. This allows you to set your audit log destination (the endpoint URL for your SIEM provider) and reuse it. 
+   * **Region endpoint**: The external endpoint that will receive audit log messages. 
+   * **Authorization Header**: The authorization type and credential to pass to your log collection endpoint. 
+    {{site.konnect_short_name}} will send this string in the `Authorization` header of requests to that endpoint.
+
+     For example, if you are setting up the webhook for Splunk, you could provide a Splunk access token: 
+     `"authorization":"Splunk example-token12234352535235"`.
+        
+    * **Log Format**: The output format of each log message. Can be CEF or JSON.
+    * **Skip SSL Verification**: Skip SSL verification of the host endpoint when delivering payloads.
+
+     {:.note}
+     > We strongly recommend not setting this to `true` as you are subject to man-in-the-middle and other attacks. This option should be considered only when using self-signed SSL certificates in a non-production environment.
+1. From the navigation menu, open {% konnect_icon Dev-portal %} **Settings**, then **Audit Logs Setup**.
+1. Fill in the fields in the **Setup** tab.
```

Comment:
@smritikjaggi I haven't revised the Dev Portal audit log UI instructions yet since I can't run through the flow yet until internal release. I think this will eventually be **Organization** > **Audit Log Setup** > **Webhook Destination** so they can configure the audit log destination first before going to the **Dev Portal** tab in the audit log setup.